### ISSUE TITLE
Feature request: Upgrading Implot Wrapper to a more recent version

### ISSUE THREAD

**Issue by AUK1939** [JuliaImGui/ImPlot.jl#33]
Great work! This is very useful. 

I was wondering if there are any plans to move to implot v0.16 or at least v0.9+?

Thanks

**Comment by JamesWrigley** [comment]
The package version number is only for the Julia package, looking at the `CImGuiPack_jll` version it looks like this is currently built for implot v0.14.

**Comment by AUK1939** [comment]
Thanks, does this mean the comment in implot_demo.jl is no longer valid?

```
#= PlotCandlestick requires imgui_internal.h, which is available in cimplot v0.9+ only
# Example for Custom Plotters and Tooltips section.#
```

**Comment by JamesWrigley** [comment]
Yes that should work now, though TBH from a quick glance I don't see anything from `imgui_internal` in there anyway.

**Comment by JamesWrigley** [comment]
v0.5 uses implot v0.16 + few more recent commits.
